python/tskit/trees.py

```diff
@@ -3041,11 +3040,13 @@ def simplify(
             sequence.
         :rtype: .TreeSequence or a (.TreeSequence, numpy.array) tuple
         """
-        tables = self.dump_tables()
+        tb = self.dump_tables()
```

Comment:
have to use 'tb' not 'tables' as the variable so it doesn't get confused with the imported tskit.tables